test/turbopack-tests-manifest.json

```diff
@@ -1297,7 +1297,7 @@
       "ReactRefreshLogBox default unterminated JSX",
       "ReactRefreshLogBox turbo internal package errors"
     ],
-    "flakey": [],
+    "flakey": ["ReactRefreshLogBox turbo conversion to class component (1)"],
```

Comment:
@wbinnssmith you need to update the `build-turbopack-manifest script so this isn't reverted again by the automatic update